ffi/tests/ffi_export.rs

```diff
@@ -54,7 +54,7 @@ impl FfiStruct {
 
     /// With params
     #[must_use]
-    pub fn with_params(mut self, params: impl IntoIterator<Item = (Name, Value)>) -> Self {
+    pub fn with_params(mut self, params: impl IntoIterator<Item = (Name, Value)>) -> FfiStruct {
```

Comment:
```suggestion
    pub fn with_params(mut self, params: impl IntoIterator<Item = (Name, Value)>) -> Self {
```

why the change? I don't mind if it doesn't affect anything

Reply:
@mversic, i added this change to showcase that such style of writing signature are supported now

Reply:
Add comment in code?